File: PharmacyApp/src/main/java/com/pharmacy/model/Reply.java
package com.pharmacy.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "replies")
public class Reply {
	
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
	public Integer id;
	public String objectionId;
	public String content;
	
	public Reply() {}
	
	public Reply(int id, String objectionId, String content) {
		super();
		this.id = id;
		this.objectionId = objectionId;
		this.content = content;
	}

	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getObjectionId() {
		return objectionId;
	}
	public void setObjectionId(String objectionId) {
		this.objectionId = objectionId;
	}
	public String getContent() {
		return content;
	}
	public void setContent(String content) {
		this.content = content;
	}
	
	
}

File: PharmacyApplication/Pharmacy/Model/Reply.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Pharmacy.Model
{
    public class Reply
    {
        public int Id { get; set; }
        public string ObjectionId { get; set; }
        public string Content { get; set; }

        public Reply() { }

        public Reply(int id, string objectionId, string content)
        {
            Id = id;
            ObjectionId = objectionId;
            Content = content;
        }
    }
}
